Sources/Context.swift

```diff
@@ -22,7 +22,8 @@ public enum StencilContext {
   /// - Throws: `Parameters.Error`
   public static func enrich(context: [String: Any],
                             parameters: [String],
-                            environment: [String: String] = ProcessInfo().environment) throws -> [String: Any] {
+                            environment: [String: String] =
+                              ProcessInfo.processInfo.environment) throws -> [String: Any] {
```

Comment:
That's a SwiftLint bug. I've filed a PR to fix it here: realm/SwiftLint#1726

However, since that patch won't be available until a future SwiftLint release, I'll adapt my code to pass linting with the current release of SwiftLint.